src/routes/edit/[slug].svelte

```diff
@@ -46,7 +47,10 @@
 	async function submitEntry() {
 		// make a put request to /api/entry/<id>.json
 		// if successful, redirect to /entry/<slug>
-
+		entryTags = entryTags.split(',')
+		if (entryTags.length == 1 && entryTags[0] == '') {
+			entryTags = []
+		}
```

Comment:
This currently doesn't check for spaces around tags, also imo the filter is nicer than an if statement.
```suggestion
		entryTags = entryTags.split(',').map(tag => tag.trim()).filter(tag => tag.length > 0)
```